_pages/team.md

```diff
@@ -46,13 +46,13 @@ He is multidisciplinary trained in Pre-clinical Medicine, Neural Engineering, Me
 ### Technology Evangelist-Pharma and Convenor
 #### @srija-ieee
 
-Dr Srija is the Co-Founder of AyeAI Consulting and Heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
+Dr Srija is the Co-Founder of AyeAI Consulting and heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
 
-She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a handson session on topic “AI in Healthcare” Dr Srija is familiar with Course design, Seminars and Workshop delivery. 
+She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a hands-on session on topic “AI in Healthcare.” Dr Srija is also familiar with Course design, Seminars and Workshop delivery. 
 
-Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in Biz ops like Regulatory Compliance, Banking Operations, HR management, Public Relations, Business development Collaterals design, Awareness and Marketing Campaign design and other technical aspects like Gitops, Website building etc.. She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research 6 TH Latin America Conference, Brazil - 2017 and 20th Annual European Congress.-2017.
+Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in biz-ops like Regulatory Compliance, Banking Operations, HR Management, Public Relations, Business Development, Collaterals design, Awareness and Marketing Campaign Design and other technical aspects like GitOps, Website development among others, She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research - 6th Latin America Conference, Brazil - 2017 and 20th Annual European Congress - 2017.
```

Comment:
D in Collaterals Design is still missed out in this commit

Reply:
done